### PR TITLE
setup mobile version skeleton

### PR DESCRIPTION
**Pull request by freddykaberuka**
## Completed Task

1.  add index page
2.  add app css
3.  add assets of image
4.  create mobile version of portofolio

### Screenshot
microverse-mod1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            
+            <div class="personal-social-info">
+                <h3>Let's CONNECT</h3>
+                <div class="social-media">
+                    <ul>
+                    <li><a href="#"><img src="/assets/twitter.svg" alt="twitter"/></a></li>
+                    <li><a href="#"><img src="/assets/in.png" alt="linkdin"></a></li>
+                    <li><a href="#"><img src="/assets/m.svg" alt=""></a></li>
+                    <li><a href="#"><img src="/assets/git.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="/assets/peace.svg" alt="peace"></a></li>
```
Comment (CollinsTatang) [inline index.html]: - [x] Please fix your image path `src="/assets/twitter.svg" ` it doesn't display

### **Correct Path**
`src="assets/twitter.svg" `

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css">
+    <title>Freddy KABERUKA</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="top-nav">
+            <div class="logo">
+                <h1>FREDDY K</h1>
+            </div>
```
Comment (CollinsTatang) [inline index.html]: - [x] Please put your logo inside a link tag `<a>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    <path d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z" />
+                </svg>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="personal-info">
+            <div class="personal-info-wrap">
+                <div class="personal-info-title">
+              <h2><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h2>
```
Comment (CollinsTatang) [inline index.html]: - [x] Kindly put your main title in `h1` tag

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @freddykaberuka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by freddykaberuka** [comment]
@CollinsTatang ,all fixed. thanks

File: index.html [hunk 4]
```diff
@@ -24,7 +24,7 @@ <h1>FREDDY K</h1>
         <section class="personal-info">
             <div class="personal-info-wrap">
                 <div class="personal-info-title">
-              <h2><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h2>
+              <h1><a href="#" class="header-info">I'm Freddy Glad to see you!</a></h1>
```
Comment (xtrahuman) [inline index.html:27]: [optional] a tag is used to anchor links, based on UX it isn't necessary to wrap this heading with a tag.

**Review by xtrahuman — APPROVED** [review]
# Approved ✔️ 
Hello @freddykaberuka, 

You have implemented all the changes requested by the previous reviewer.

Your project is complete!  you have done a great job 😃 ... it's time to merge it :shipit: 
Congratulations! 🎉 

However, I dropped some optional suggestions to improve your project below 👇 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**